cli/src/main/java/com/liferay/blade/cli/command/DeployCommand.java

```diff
@@ -59,7 +60,24 @@ public void execute() throws Exception {
 		int port = 11311;
 
 		if (!BladeUtil.canConnect(host, port)) {
-			_addError("deploy", "Unable to connect to gogo shell on " + host + ":" + port);
+			StringBuilder sb = new StringBuilder();
+
+			sb.append("Unable to connect to gogo shell on " + host + ":" + port);
+			sb.append(System.lineSeparator());
+			sb.append("Liferay may not be running, or the gogo shell may need to be enabled");
+			sb.append(System.lineSeparator());
+			sb.append("Please see this link for more details:");
+			sb.append(System.lineSeparator());
+			sb.append("https://issues.liferay.com/browse/LPS-82849");
```

Comment:
can we link to documentation on how to set up gogo shell instead of a link to an LPS ticket

Reply:
https://dev.liferay.com/en/develop/tutorials/-/knowledge_base/7-1/using-developer-mode-with-themes#setting-developer-mode-for-your-server-using-portal-developer-properties